src/content/release-notes.mdx

```diff
@@ -5,6 +5,42 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.27.0
+
+9 December 2024
+
+This version is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- Remove code for our deprecated Quickstarts feature <!-- 8842 -->
+
+### New Features
+
+- We now allow the ability for admins of Okteto to [set the minimum supported CLI version for their team](admin/dashboard.mdx#command-line-cli)  <!-- 8829 -->
+- We've published the Okteto Manifest JSON Schema for inline suggestions and validation for creating and editing Okteto manifests within your code editor
+- Added support for overwriting CLI images at the Helm chart level
+- Shipped additional support for the installation of [Okteto in Air-Gapped Environments](self-hosted/manage/air-gapped.mdx) <!-- 8914 -->
+- Configured buildkit probes to only accept requests when buildkit is healthy  <!-- 8871 -->
+- Added support for configuring the Okteto control plane jobs TTL at the helm level  <!-- 8917 -->
+- feat: send cli image on graphql clustermetadata  <!-- 8914 -->
+
+### Improvements
```

Comment:
We should mention also the `3.2` release for the CLI. relevant things to mention:
* CLI image (`okteto/okteto`) is now rootless
* You can define a variable `OKTETO_DEV_PERSISTENT_VOLUME_SIZE`, and it will change the default volume size for dev containers (it only applies if the manifest doesn't set specifically a size)
* Fix: Fixed a problem when deploying a compose with failed healthchecks. We were not taking into account the timeout operation, so the deploy operation was stuck forever

Reply:
After the latest changes and the discussion we had on Monday, we should remove the point about `okteto/okteto` image being rootless. We have reverted it